Description: [Boost.Test] g++ compilation error due to ambiguity 
 Applied upstream changeset 76600 and additional change by t0rt1e@… 
Author: rogeeff, t0rt1e@… 
Bug: 671409
Forwarded: https://svn.boost.org/trac/boost/ticket/6331


Index: boost/test/impl/exception_safety.ipp
===================================================================
--- a/boost/test/impl/exception_safety.ipp	(revision 76599)
+++ b/boost/test/impl/exception_safety.ipp	(revision 76600)
@@ -51,10 +51,10 @@
 
 namespace boost {
 
-using namespace ::boost::unit_test;
- 
 namespace itest {
 
+using namespace ::boost::unit_test;
+
 // ************************************************************************** //
 // **************             execution_path_point             ************** //
 // ************************************************************************** //

Index: boost/test/impl/logged_expectations.ipp
===================================================================
--- a/boost/test/impl/logged_expectations.ipp (Revision 76602)
+++ b/boost/test/impl/logged_expectations.ipp (Arbeitskopie)
@@ -40,10 +40,10 @@
 
 namespace boost {
 
+namespace itest {
+
 using namespace ::boost::unit_test;
 
-namespace itest {
-
 // ************************************************************************** //
 // **************    logged expectation test implementation    ************** //
 // ************************************************************************** //
